astropy/coordinates/tests/test_representation_arithmetic.py

```diff
@@ -1407,7 +1407,7 @@ def test_operation_unit_spherical(self, diff_cls, op, args):
         # and division there will be sign flips in the spherical distance.
         expected_c = op(
             rep.represent_as(CartesianRepresentation, {"s": CartesianDifferential}),
-            *args
+            *args,
```

Comment:
Can you make these changes as part of a separate PR?

Reply:
Unfortunately, black decided these changes were necessary after I made these changes.

I am investigating why this might be.

Reply:
Further investigation has revealed to me that by just migrating the `[metadata]` entries from `setup.cfg` to `[project]` in the `pyproject.toml` seems to trigger `black` wanting to inject all these commas.

I have determined that the changes made by `black` triggered by this change are totally fine under without the `pyproject.toml` migration. Thus I think the simplest course of action is to merge those changes first and then merge the meaningful changes proposed here.